sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -46,8 +46,17 @@ if (NOT DEFINED L0_LIBRARY OR NOT DEFINED L0_INCLUDE_DIR)
   list(APPEND SYCL_TOOLCHAIN_DEPLOY_COMPONENTS l0-loader)
 else()
   include_directories("${L0_INCLUDE_DIR}")
-  add_custom_target(l0-loader
-    DEPENDS ${L0_LIBRARY} COMMENT "Using provided L0 Loader")
+  include_directories("${L0_INCLUDE_DIR}")
+  file(GLOB L0_LIBRARY_SRC "${L0_LIBRARY}*")
+  file(COPY ${L0_LIBRARY_SRC} DESTINATION ${LLVM_LIBRARY_OUTPUT_INTDIR})
+  add_custom_target(l0-loader DEPENDS ${L0_LIBRARY} COMMENT "Copying Level Zero Loader ...")
+  if(MSVC)
```

Comment:
Please remind me why MSVC uses static library?

Reply:
It is not a static library, it is an import library for .dll. To use the dll, application should link with this import library. It just has the same .lib extension as static libraries.

Reply:
OK, then I am surprised that `cmake` doesn't have something for it. Linking is always to .lib on Windows and .so on Linux. Even if there is a better way to write this, it shouldn't block this PR.